DOCS.md

```diff
@@ -63,6 +63,38 @@ drone secret add \
 
 Kubernetes expects secrets to be base64 encoded, `drone-gke` does that for you. If you pass in a secret that is already base64 encoded, please apply the prefix `secret_base64_` and the plugin will not re-encode them.
 
+## Available vars
+
+These variables are always available to reference in any manifest, and cannot be overwritten by `vars` or `secrets`:
+
+```
+---START VARIABLES AVAILABLE FOR ALL TEMPLATES---
+{
+	"BRANCH": "master",
+	"BUILD_NUMBER": "12",
+	"COMMIT": "4923x0c3380413ec9288e3c0bfbf534b0f18fed1",
+	"TAG": "",
+	"cluster": "my-gke-cluster",
+	"namespace": "",
+	"project": "my-gcp-proj",
+	"zone": "us-east1-a"
+}
+---END VARIABLES AVAILABLE FOR ALL TEMPLATES---
+```
+
+## Expanding environment variables
+
+It may be desired to reference an environment variable for use in the Kubernetes manifest.
```

Comment:
in the example below, it's being used in the drone config

Reply:
I'm not sure what you mean by this.

If you want to use an environment variable in a manifest, you have to pass it to the manifest as a variable in `vars`.